src/Nest/Mapping/Types/Core/Range/Ip/IpRangeAttribute.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	public class IpRangeAttribute : RangePropertyAttributeBase, IIpRangeProperty
```

Comment:
`IpAddressRangeAttribute`, to align with `IpAddressRange`?

Reply:
:-1: